Lombiq.NodeJs.Extensions/Lombiq.NodeJs.Extensions.targets

```diff
@@ -156,6 +169,7 @@
     <ItemGroup>
       <NodeJsExtensionsTargetFiles
         Include="%(NodeJsExtensionsTargetDirectories.Identity)/**/*"
+        Exclude="%(NodeJsExtensionsTargetDirectories.Identity)/**/_temp_/**/*"
```

Comment:
I've added this to fix an intermittent error that sometimes showed up, and it happened to me again, today:

![image](https://user-images.githubusercontent.com/164360/221898596-595ea6fb-433a-4c52-b02c-457e0843a2d8.png)